src/Models/Category.php

```diff
@@ -12,6 +12,11 @@ class Category extends Model
 {
     use HasAlternatesThroughRewrites;
 
+    protected $casts = [
+        self::UPDATED_AT => 'datetime',
```

Comment:
note: i'm using self instead of static as static cannot be used since it's created at runtime and $casts is compiled earlier.